ui/packages/consul-ui/tests/steps.js

```diff
@@ -87,6 +87,7 @@ export default function({
     api.server.respondWith(url.split('?')[0], data);
   };
   const setCookie = function(key, value) {
+    document.cookie = `${key}=${value}`;
```

Comment:
Will we share this cookie state between tests? do we need to reset this cookie after the test is complete again?

Reply:
Nope no sharing between tests. All cookies are reset in a beforeEach hook